test/patterns/classic_setter/test_classic_setter.py

```diff
@@ -0,0 +1,22 @@
+
+import os
+from unittest import TestCase
+from aibolit.patterns.classic_setter.classic_setter import ClassicSetter
+from pathlib import Path
+
+
+class TestSetter(TestCase):
+    dir_path = Path(os.path.realpath(__file__)).parent
+    setter_finder = ClassicSetter()
+
+    def test_one_valid_patterns(self):
+        lines = self.setter_finder.value(Path(self.dir_path, 'BaseKeyframeAnimation.java'))
+        assert lines == [40]
```

Comment:
Please, stop using "bare" `assert` statement, and use `unittest.TestCase` [assertion methods](https://docs.python.org/3/library/unittest.html#unittest.TestCase) instead.